### PR TITLE
Avatar

### LINKED ISSUE #11: Avatar
## Criação do Avatar:
- Objetivo: Criar o componente para importar a imagem do cliente
## Dados do Ambiente:
- Biblioteca de componentes: https://ui.shadcn.com/docs/components
- Usar essa descrição para o componente: componentName.jsx
### Para rodar o storybook: npx storybook@latest init
- Usar essa descrição para o storybook: componentName.stories.tsx. 
Exemplo:
image

## Comportamento Esperado: 
- Aparecer o ícone com a foto do usuário no formato redondo.
## Comportamento Real: 
- Carregar a foto do usuário (quando carregada) ou as iniciais (caso não exista foto ou dê erro no carregamento da imagem) dentro de um formato redondo (border-radius 50%).

## Imagens/Capturas de Tela:
image

### Registros de Erro: 
< Os famosos logs, se disponíveis, podem fornecer informações valiosas sobre o problema. >

### Possível Solução/Sugestões: 
```
<fieldset class="half username-fieldset">
   <div class="account-avatar">
      <label for="avatar" class="label">Avatar</label>
      <div class="account-avatar__field-wrap">
         <img src="https://gravatar.com/avatar/iniciais-do-nome.png" alt="User Avatar">
         <input class="account-avatar__field" data-user-id="user-id" type="file" id="avatar">
         <small class="field-desc">Maximum size: 10MB. Minimum dimensions: 500x500px.</small>
      </div>
   </div>
</fieldset>
```

### PR DESCRIPTION
**Pull request by MansuetoPaiva**
Closes #11 
https://github.com/dcTeam23/cat-list/pull/11#issuecomment-2094256648

online at: https://6634f18d79a7835323a8328c-hpywbppsss.chromatic.com/?path=/story/components-avatar--primary&args=format:circle

### CODE REVIEW

**Comment by AbraaoAlves** [comment]
If you want a review of this PR, please follow these steps:

@MansuetoPaiva 
https://github.com/dcTeam23/cat-list/pull/15#issuecomment-2094256648

File: src/components/Avatar/Avatar.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          </div>
+    );
+  }  if (format === "circle") {
+    return (
+      <div className="container flex justify-center items-center">
+    <img src={imgSrc} className={"avatar circle w-32 h-32 rounded-full object-cover" + format} alt={descricao} />
+    </div>
+    );
+  }
+}
```
Comment (AbraaoAlves) [inline src/components/Avatar/Avatar.jsx:27]:
``` suggestion

    return (
      <div className="container flex justify-center items-center">
        <img src={imgSrc} className={`avatar ${format} w-32 h-32 border-3 border-white object-cover`} alt={descricao} />    
          </div>
    );
}
```

Comment (MansuetoPaiva) [reply src/components/Avatar/Avatar.jsx]: preciso de ajuda para ajustar o tailwind para essa sugestão

Comment (MansuetoPaiva) [reply src/components/Avatar/Avatar.jsx]: consegui com a ajuda do @joel

**Review by AbraaoAlves — DISMISSED** [review]

**Review by AbraaoAlves — APPROVED** [review]

**Pull request merged**